eventuate-messaging-kafka-basic-consumer/src/main/java/io/eventuate/messaging/kafka/basic/consumer/EventuateKafkaConsumer.java

```diff
@@ -27,6 +29,9 @@
   private AtomicBoolean stopFlag = new AtomicBoolean(false);
   private Properties consumerProperties;
   private volatile EventuateKafkaConsumerState state = EventuateKafkaConsumerState.CREATED;
+  private KafkaConsumer<String, String> consumer;
```

Comment:
Why is this a field instead of a local variable?

Reply:
Fixed